scrapers/vip4k.yml

```
@@ -0,0 +1,49 @@
+name: vip4k
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - black4k.com
+      - daddy4k.com
+      - fist4k.com
+      - hunt4k.com
+      - mature4k.com
+      - old4k.com
+      - rim4k.com
+      - sis.porn
+      - stuck4k.com
+      - tutor4k.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title: //div[@class="title_player" or @class="title"]| //h1[contains(@class,"player")][contains(@class,"title")]
+     Performers:
+      Name:
+        selector: //a[@class="link_player"]/text()
```

the selector for sis.porn and the site that have the performers with the `star` icon was missing
```suggestion
      Performers:
        Name:
          selector:
            //a[@class="link_player"]|//div[@class="player-item__row"][contains(div,"Name:")]//span|
            //div[contains(@class,"player-item")]//*[local-name()="svg"][@class="ico ico--star"]/../following-sibling::div
```